src/cpu_feature/cpu_feature.rs

```diff
@@ -0,0 +1,61 @@
+extern crate libc;
+
+#[cfg(all(target_os = "linux", target_arch = "arm"))]
+use std::fs::File;
+#[cfg(all(target_os = "linux", target_arch = "arm"))]
+use std::io::BufReader;
+// TODO represent unsigned long without libc
+#[cfg(all(target_os = "linux", target_arch = "arm"))]
+use self::libc::c_ulong;
+#[cfg(all(target_os = "linux", target_arch = "arm"))]
+use self::auxv::{AuxValError, AuxVals};
+#[cfg(all(target_os = "linux", target_arch = "arm"))]
+use self::cpuinfo::CpuInfo;
+#[cfg(all(target_os = "linux", target_arch = "arm"))]
+use self::arm::arm_cpuid_setup;
+
+#[cfg(all(target_os = "linux", target_arch = "arm"))]
+static HAS_BROKEN_NEON: bool = false;
```

Comment:
I think you're going to remove this (see below), but note that this would need to be `static mut` so that it can be properly set.